models/project/src/project/transformers.ts

```diff
@@ -0,0 +1,43 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TProject, TProjectGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TProject, TProject>('default', {
+    buildFields: [
```

Comment:
**nit**: we can extract `buildFields` to a constant and reuse them in the different transformers.